terraform/main/openstack/inputs.tf

```
@@ -0,0 +1,73 @@
+locals {
+  project_name = "lgr"
```

```suggestion
  project_name = "st"
```

Rationale: we standardized to "st" for all prefixes by default. You can make it an overridable variable if that helps.